docs/machine-learning/ml-problem-formulation-envisioning.md

```diff
@@ -34,6 +35,19 @@ or disproved - to guide the exploration of the data science team. Where possible
 
 Once an understanding of the problem domain has been established, it may be necessary to break down the overall problem into smaller, meaningful chunks of work to maintain team focus and ensure a realistic project scope within the given time frame.
 
+## Listening to the customer
+
+Throughout each session, we must keep in mind and proactively practice the exercise of listening to our customer. They are the center of gravity of any engagement, so we need to practice compassion by stepping into their shoes to feel their pain point. Therefore, it is important to always be mindful of their thoughts by asking leading questions to understand:
+
+* Who is the end user?
+* What is the current practice related to the business problem?
+* What's the performance of the current framework?
+* What are their pain points?
+* What is their toughest problem?
+* What is the state of the data used to build the solution?
+
+It is not uncommon for the party who reached out CSE to not be the end user of the framework. In these cases, listening to the actual end users is critical to the success of the project.
```

Comment:
I really like the point you are making here, but I think it should be generalised. 

Essentially your recommendation is that you need to speak to the end users of the system. Given how important this point is, I would move it to the start of the section. Perhaps it could be combined with the first line if you define the customer to be the end users.

Reply:
Agree with Bianca's and Chris's points. This could be generalized to look at the end user as the customer.